.gitignore

```diff
@@ -1,6 +1,9 @@
 *.py[co]
 *~
+examples/.ipynb_checkpoints/
```

Comment:
Would something like `*/.ipynb_checkpoints/` match both checkpoint directories in the `example/` folder and the top level? That would be sweet, mainly because I am messy and start `jupyter` in either directory :-/